src/foam/nanos/auth/Address.js

```diff
@@ -65,13 +68,30 @@ foam.CLASS({
       class: 'Reference',
       targetDAOKey: 'countryDAO',
       name: 'countryId',
-      of: 'foam.nanos.auth.Country'
+      of: 'foam.nanos.auth.Country',
+      view: function(_, X) {
```

Comment:
I think this could be done more simply as just:
view: { class: 'foam.u2.view.ChoiceView', objToChoice: function(a) { return [a.id, a.name]; } }